README.md

```diff
@@ -51,10 +51,19 @@ $ yarn clean
 ```
 
 ## Run local version of Burnr wallet
-Running the following command will build all necessary dependencies and run the Substrate Burnr Wallet in development mode with hot reloading enabled.
+Running the following command will build all necessary dependencies and run the Substrate Burnr Wallet in development mode with hot reloading enabled. It will be served on http://localhost:8000/
 
 ```
-$ yarn run dev
+$ yarn run dev:burnr
+```
+
+(Make sure to run `$ yarn install` before.)
+
+## Run local version of the Soldot browser demo
+Running the following command will build all necessary dependencies and run the Soldot browser demo.   It will be served on http://localhost:1234/
```

Comment:
Typo in heading and instructions "Smoldot" -> "Smoldot".  And runs on SSL, so `https://localhost:1234`